### PR TITLE
modification for removing leftmost questions truncate option

### PR DESCRIPTION
**Pull request by aarora8**
- Removed --leftmost-questions-truncate option from 
steps/chain/build_tree.sh and src/bin/compile-questions.cc. 
 - Removed the usage of this option from "kaldi/egs/*/*/local/chain/*.sh" and  "kaldi/egs/*/*/local/nnet3/*.sh".

### CODE REVIEW

File: egs/wsj/s5/steps/nnet3/chain/build_tree.sh [hunk 1]
```diff
@@ -22,9 +22,6 @@ cmd=run.pl
 context_opts=  # e.g. set this to "--context-width 5 --central-position 2" for quinphone.
 cluster_thresh=-1  # for build-tree control final bottom-up clustering of leaves
 frame_subsampling_factor=1
-leftmost_questions_truncate=-1  # note: this used to default to 10, but we never
-                                # use this option now with value != -1, and
-                                # we're changing the default
 tree_stats_opts=
```
Comment (hhadian) [inline egs/wsj/s5/steps/nnet3/chain/build_tree.sh]: I guess you shouldn't remove it from here so that the old scripts calling this script won't crash.

**Comment by danpovey** [comment]
@hhadian, that probably makes sense but the comment (and  if applicable,
usage message) should be changed to say that it's deprecated and has no
effect.

File: egs/wsj/s5/steps/nnet3/chain/build_tree.sh (hunk 1 above)
Comment (aarora8) [inline egs/wsj/s5/steps/nnet3/chain/build_tree.sh]: Pushed with changes.

File: egs/wsj/s5/steps/nnet3/chain/build_tree.sh [hunk 2]
```diff
@@ -158,17 +156,13 @@ if [ $stage -le -3 ] && $train_tree; then
...
   cat $lang/phones/extra_questions.int >> $dir/questions.int
   $cmd $dir/log/compile_questions.log \
-    compile-questions --leftmost-questions-truncate=$leftmost_questions_truncate \
-      $context_opts $lang/topo $dir/questions.int $dir/questions.qst || exit 1;
+    compile-questions $context_opts $lang/topo \
+      $dir/questions.int $dir/questions.qst || exit 1;
 
   # questions_truncated.int will be needed later on when we build the phone
   # language model for 'chain' training.  It's a mechanism of keeping the graph
   # small.
```
Comment (hhadian) [inline egs/wsj/s5/steps/nnet3/chain/build_tree.sh]: I guess `questions_truncated.int` is never used so this comment and the following `cp ...` line could be removed but not sure.

Comment (aarora8) [inline egs/wsj/s5/steps/nnet3/chain/build_tree.sh]: Removed, questions_truncated.int and comment.

**Comment by danpovey** [comment]
You added a file
egs/wsj/s5/steps/nnet3/chain/.build_tree.sh.swp
to the repository!  Please remove it.

**Comment by aarora8** [comment]
removed build_tree.sh.swp file

**Pull request merged**